### PR TITLE
add xyz2mol

### PR DESCRIPTION
**Pull request by hadim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hadim** [comment]
@jhjensen2: here the recipe that will build the conda package. Once merged, a feedstock repo will be created where the package will be built and uploaded. Once the process done (after a review step from a conda forge dev), the package will be available at `conda install -c conda-forge xyz2mol`). For more details see the documentation at https://conda-forge.org/docs/index.html

Please explicitly confirm here you agree to be a maintainer of the package (this is mandatory before merging).

**Comment by hadim** [comment]
@conda-forge/staged-recipes please review. I am not sure to understand the windows CI error, but it might be irrelevant since the package is pure python. Let me know.

File: recipes/xyz2mol/meta.yaml
Comment (dopplershift) [inline recipes/xyz2mol/meta.yaml:15]: This should fix Windows:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  entry_points:
    - xyz2mol=xyz2mol:main
```

Comment (hadim) [reply recipes/xyz2mol/meta.yaml:15]: I wasn't aware of that trick. Thanks!

**Comment by jhjensen2** [comment]
I agree to be a maintainer on the package

**Comment by hadim** [comment]
@dopplershift LGTM here

**Review by dopplershift — APPROVED** [review]

**Pull request merged**
